src/ucp/rndv/proto_rndv.c

```diff
@@ -462,6 +462,14 @@ void ucp_proto_rndv_rts_query(const ucp_proto_query_params_t *params,
 void ucp_proto_rndv_rts_abort(ucp_request_t *req, ucs_status_t status)
 {
     ucp_am_release_user_header(req);
+
+    if (ucp_request_invalidation_required(req)) {
```

Comment:
```c
if (ucp_request_invalidation_required(req)) {
   ucp_request_datatype_iter_invalidate(req, status);
}
```

Reply:
We cannot do that because in case of memory invalidation we call `ucp_request_complete_send` later from  `ucp_request_memh_invalidate_progress` func call that will be called from `callback_q`.